extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -1449,24 +1449,29 @@ export class AmpStoryPage extends AMP.BaseElement {
    * @private
    */
   checkPageHasAudio_() {
-    const pageHasAudio =
-      this.element.hasAttribute('background-audio') ||
-      this.element.querySelector('amp-audio') ||
-      this.hasVideoWithAudio_();
+    this.hasVideoWithAudio_().then((hasVideoWithAudio) => {
+      const pageHasAudio =
+        this.element.hasAttribute('background-audio') ||
+        this.element.querySelector('amp-audio') ||
+        hasVideoWithAudio;
```

Comment:
Nit: Plz put this one first so we don't run the other statements when it's `true`